### PR TITLE
Release/0.10.0 a/v

### PR DESCRIPTION
**Pull request by E1337Kat**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+
+As always, all feedback is appreciated so we can ensure that no bugs or oversights have slipped through our testing — and any suggestions and ideas are welcome too! Please get in touch either on Discord, Github, or Nexus (links at the bottom).
+
+## Main Jobs
+
+- ✨ **Modlist File Support** The latest cyberpunk update added a feature to redmod to allow deploying mods specified in a file. When we read that, we knew we had to get right on it since many people have been running into a limit on mods because of how long the deploy command is. This will fix that right up.
```
Comment (effs) [inline CHANGELOG.md:13]: Highlight the user-facing “No more mod count limit on REDmods (Eat it NetWatch!)” or so?

File: src/load_order.ts [hunk 2]
```diff
@@ -586,7 +586,8 @@ export const startREDmodDeployInTheBackgroundWithNotifications = (
   const REDdeployment: Promise<void> =
     pipe(
       loadOrderToREDdeployModList(v2077LoadOrderToDeploy),
-      (generatedModList) => generatedModList.join(`\n`),
+      // The line joining MUST be \r\n so that it is Windows line endings. otherwise redmod fails
+      (generatedModList) => generatedModList.join(`\r\n`),
```
Comment (effs) [inline src/load_order.ts:590]: Maybe overdefensive but [`os.EOL`](https://nodejs.org/api/os.html#oseol)

**Review by effs — APPROVED** [review]
2 bits, good to go otherwise!

File: CHANGELOG.md (hunk 1 above)
Comment (E1337Kat) [reply CHANGELOG.md:13]: I like that. Any ideas for the release copy? It's fine if not.

**Pull request merged**